SwanGallery/SwanGallery/handlers.py

```diff
@@ -0,0 +1,200 @@
+import json
```

Comment:
If I understand correctly, these python files are simply copied from `SwanContents` right?

Once we refactor/figure out how to use `SwanContents` locally, we can get rid of these files right?


If not, i.e if there is some extra logic added on top, can you add a comment on the top of the file and describe what it changes